pkg/node/node_labels.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Authors of Cilium
```

Comment:
Small nit the file could be called `labels.go`

Reply:
Ah yeah, that makes more sense.

I named it after `node_address_xxx.go`. Should I rename those as well?

Reply:
I guess it would make sense yes.